Documentation/helm-operator.md

```diff
@@ -106,7 +106,7 @@ The following tables lists the configurable parameters of the rook-operator char
 | `csi.provisionerPriorityClassName`  | PriorityClassName to be set on csi driver provisioner pods.                                                                 | <none>                                                    |
 | `csi.enableOMAPGenerator`           | EnableOMAP generator deploys omap sidecar in CSI provisioner pod, to enable it set it to true                               | `false`                                                   |
 | `csi.rbdFSGroupPolicy`              | Policy for modifying a volume's ownership or permissions when the RBD PVC is being mounted                                  | ReadWriteOnceWithFSType                                   |
-| `csi.cephFSFSGroupPolicy`           | Policy for modifying a volume's ownership or permissions when the CephFS PVC is being mounted                               | ReadWriteOnceWithFSType                                   |
+| `csi.cephFSFSGroupPolicy`           | Policy for modifying a volume's ownership or permissions when the CephFS PVC is being mounted                               | `None`                                  |
```

Comment:
Is this only for cephfs not for rbd?

Reply:
Yeah, atleast in this stage, I see only `cephfs` as the possible candidate here, the reason being the global permission set we do and also due to the fact of `rwx` share type volume. Have to think some more before proposing the change for `rbd` and especially while it support both `block and filemode`.